.CI/build_all

```diff
@@ -25,4 +27,4 @@ git ls-tree --name-only master -- . | xargs -I {} sh -c "rm -rf {} && echo Remov
 popd > /dev/null
 
 # We just want to build all of the recipes.
-conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5"
+conda-build-all ./recipes --matrix-condition "numpy >=1.11" "python >=2.7,<3|>=3.5" "r-base >=3.3.2"
```

Comment:
To be clear, I'm totally ok with these changes for now. We may need to hack something in the feedstock as well. I'd prefer though if we can add this in a separate PR to staged-recipes so it is easier to track and remove once it is fixed. Also feel free to rebase this on said separate PR so it is ready to go and using the same commit.

Note: When no recipes are in a PR, the linter complains. It is easy to silent the linter by adding `[skip lint]` to the commit message.

xref: https://github.com/SciTools/conda-build-all/pull/74